app/bundles/CategoryBundle/CategoryEvents.php

```diff
@@ -56,4 +56,14 @@ final class CategoryEvents
      * @var string
      */
     public const CATEGORY_ON_BUNDLE_LIST_BUILD = 'mautic.category_on_bundle_list_build';
+
+    /**
+     * The mautic.category_type_entity event is thrown when get specific bundle entity.
+     *
+     * The event listener receives a
+     * Mautic\CategoryBundle\Event\CategoryTypeEntityEvent instance.
+     *
+     * @var string
+     */
+    public const CATEGORY_TYPE_ENTITY = 'mautic.category_type_entity';
```

Comment:
Let's use the new way to dispatch Symfony events where this constant is not necessary. Just dispatch the new event itself.